services/auth/source/smtp/source_authenticate.go

```diff
@@ -74,10 +74,12 @@ func (source *Source) Authenticate(user *user_model.User, userName, password str
 		LoginType:   auth_model.SMTP,
 		LoginSource: source.authSource.ID,
 		LoginName:   userName,
-		IsActive:    true,
+	}
+	overwriteDefault := &user_model.CreateUserOverwriteOptions{
+		IsActive: util.OptionalBoolTrue,
```

Comment:
OK to bypass setting.Service.RegisterEmailConfirm and setting.Service.RegisterManualConfirm here?